service/controller/v13/key/key.go

```diff
@@ -168,7 +168,7 @@ func ClusterCustomer(customObject v1alpha1.AWSConfig) string {
 }
 
 func ClusterEtcdDomain(customObject v1alpha1.AWSConfig) string {
-	return customObject.Spec.Cluster.Etcd.Domain
+	return fmt.Sprintf("%s:%d", customObject.Spec.Cluster.Etcd.Domain, customObject.Spec.Cluster.Etcd.Port)
```

Comment:
I had to fix this bug as part of my changes were already in v13 but the rest was not and I didn't want to block the release. Adding etcd port as we have different ports across providers